instruments/cryomagnetics/cryomagnetics_4g.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+Implements the cryomagnetics 4G superconducting magnet power supply
+"""
+
+# IMPORTS #####################################################################
+from __future__ import absolute_import
+
+import re
+from time import sleep
+
+import quantities as pq
+
+from instruments.abstract_instruments.cryomagnetics_instrument \
```

Comment:
nit pick but I don't really like backslashes, I'd rather see

```python
from instruments.abstract_instruments.cryomagnetics_instrument import (
    CryomagneticsInstrument
)
```

But of course, you're going to have to move `CryomagneticsInstrument` as per my other comments :)